### PR TITLE
Features/task 0

### PR DESCRIPTION
**Pull request by RadekP3**

### CODE REVIEW

File: TodoListApp/TodoList.Api.Tests/DummyTests.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace TodoList.Api.Tests
+{
+    [TestFixture]
+    public class DummyTests
+    {
+        [Test]
+        public void BasicMathTest()
+        {
+            Assert.AreEqual(5, 2 + 3);
```
Comment (PetrSvirak) [inline TodoListApp/TodoList.Api.Tests/DummyTests.cs:11]: An `Assert.True` would do as well, but this one is OK nonetheless 😃

File: TodoListApp/TodoList.Api.Tests/TodoList.Api.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      <Name>TodoList.Api</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('..\packages\MSTest.TestAdapter.1.2.0\build\net45\MSTest.TestAdapter.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\MSTest.TestAdapter.1.2.0\build\net45\MSTest.TestAdapter.props'))" />
+    <Error Condition="!Exists('..\packages\MSTest.TestAdapter.1.2.0\build\net45\MSTest.TestAdapter.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\MSTest.TestAdapter.1.2.0\build\net45\MSTest.TestAdapter.targets'))" />
```
Comment (PetrSvirak) [inline TodoListApp/TodoList.Api.Tests/TodoList.Api.Tests.csproj]: I don't like these error conditions here. They still make you reliable on `MSTests`… (see below)

File: TodoListApp/TodoList.Api.Tests/packages.config [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="1.2.0" targetFramework="net472" />
+  <package id="MSTest.TestFramework" version="1.2.0" targetFramework="net472" />
```
Comment (PetrSvirak) [inline TodoListApp/TodoList.Api.Tests/packages.config]: It is not enough with a `NuGet` to remove a reference (project) you need to remove `NuGet` itself (from the NuGet manager or console)

Comment (PetrSvirak) [reply TodoListApp/TodoList.Api.Tests/packages.config]: Preferably replace the `MSTest.TestAdapter` with `NUnit`'s test adapter, you would be able to run your `NUnit` tests even without R# this way…

**Review by PetrSvirak — CHANGES_REQUESTED** [review]
I believe the repository to be almost ready. There are just 2 details remaining. Let's fix them quickly and continue with the next task 😃

File: TodoListApp/TodoList.Api.Tests/DummyTests.cs (hunk 1 above)
Comment (RadekP3) [reply TodoListApp/TodoList.Api.Tests/DummyTests.cs:11]: 👍

File: TodoListApp/TodoList.Api.Tests/TodoList.Api.Tests.csproj (hunk 2 above)
Comment (RadekP3) [reply TodoListApp/TodoList.Api.Tests/TodoList.Api.Tests.csproj]: 👍

File: TodoListApp/TodoList.Api.Tests/packages.config (hunk 3 above)
Comment (RadekP3) [reply TodoListApp/TodoList.Api.Tests/packages.config]: 👍

File: TodoListApp/TodoList.Api.Tests/DummyTests.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace TodoList.Api.Tests
+{
+    [TestFixture]
+    public class DummyTests
+    {
+        [Test]
+        public void BasicMathTest()
+        {
+            Assert.True(5 == 2 + 3);
```
Comment (PetrSvirak) [inline TodoListApp/TodoList.Api.Tests/DummyTests.cs]: Oh, I've misled you, sorry. I mixed test frameworks and did not realize that `NUnit` does not have `Assert.True()` 😞 The constraint-based assertion model is strongly preferred nowadays and it generally makes tests more readable; please remove the https://github.com/RadekP3/kentico-onboarding-cs/pull/2/commits/f9e02ea2aa28ed71aaf67a5b5af2bd00e272aa68 commit.

Comment (RadekP3) [reply TodoListApp/TodoList.Api.Tests/DummyTests.cs]: 👍

**Review by PetrSvirak — APPROVED** [review]

**Pull request merged**